### ISSUE TITLE
Critical Error:  API didn't find any team - Skipping Account

### ISSUE THREAD

**Issue by FlokiNL** [PCJones/Ultimate-Splinterlands-Bot-V2#97]
This doesn't happen every match but very often on last beta update

Critical Error:  API Error: Response was empty
[1-9-2022 09:14:17] Critical Error: API didn't find any team - Skipping Account

**Comment by PCJones** [comment]
Hi,

can you share your card settings file please?

**Comment by FlokiNL** [comment]
## Set it to true to to activate card settings
USE_CARD_SETTINGS=false

## Example 1: The team has a winrate of 50%, you own 5/5 cards of it. The API will pretend the team has a 74% WR.
## Example 2: The team has a winrate of 50%, you own 1/4 cards of it. The API will pretend the team has a 48.5% WR (50 + (1/4 * 24) - (3 * 2.5)
## Example 3: The team has a winrate of 50%, you own 0/4 cards of it. The API will pretend the team has a 40% WR.
WINRATE_MODIFIER_OWNED_CARD_PERCENTAGE=24
FLAT_NEGATIVE_MODIFIER_PER_UNOWNED_CARD=2.5

## Example: If you finished in silver last season the modifiers above will only be applied
## once you are in silver or higher
DISABLE_OWNED_CARDS_PREFERENCE_BEFORE_CHEST_LEAGUE_RATING=true

## Example: If you finished in silver last season the bot will not prioritize the quest focus element
## before you are in silver or higher
DISABLE_FOCUS_PRIORITY_BEFORE_CHEST_LEAGUE_RATING=true

## Only play cards with this level or higher
CARD_MIN_LEVEL=1

## Always add 0 mana cards like Furious Chicken if the account has it
ADD_ZERO_MANA_CARDS=true

## If set to false the bot will never play any starter cards. Only recommended if you have a ton of cards
PLAY_STARTER_CARDS=true

## Possible options: dragon,death,fire,earth,water,life
## Example: You only want to play earth and fire elements if the bot is not fighting for the quest focus:
## PREFERRED_SUMMONER_ELEMENTS=earth,fire
PREFERRED_SUMMONER_ELEMENTS=dragon,death,fire,earth,water,life

## If the API can not find a team with your preferred summoner elements with a winrate >= this
## it will chose a team without element restrictions applied
PREFERRED_SUMMONER_ELEMENTS_WINRATE_THRESHOLD=48

## If PRIORITIZE_QUEST=true then the API will always try to play the focus element, unless it can not find
## A team with a winrate >= this
USE_FOCUS_ELEMENT_WINRATE_THRESHOLD=55

####################################################################################
##### DEPRECATED - only working for old API and will probably be removed soon:

## The minimum amount of historic games the api should have for a team to consider it
MINIMUM_GAMES=10

## Possible options: dragon,death,fire,earth,water,life
PREFERRED_SUMMONER_ELEMENT=

## If the api only finds teams with a winrate smaller than this it will ignore the card settings
## (0-100)
WINRATE_THRESHOLD=40

**Comment by PCJones** [comment]
Could you check if it's fixed now?

**Comment by FlokiNL** [comment]
Yes! It’s fixed, didn’t have this error for a while now 😊 bot performance (wins) is also a lot better since last season  Verzonden vanuit Mail voor Windows Van: Jonas FVerzonden: woensdag 14 september 2022 11:47Aan: PCJones/Ultimate-Splinterlands-Bot-V2CC: FlokiNL; AuthorOnderwerp: Re: [PCJones/Ultimate-Splinterlands-Bot-V2] Critical Error: API didn't find any team - Skipping Account (Issue #97) Could you check if it's fixed now?—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

**Comment by psydrake** [comment]
First of all, thank you @PCJones for providing this software - I appreciate the work you put into it!
Unfortunately I am also seeing this issue intermittently:
```
Finding match...
Splinterlands Response: {"success":true,"id":"sl_44cc999dddb9f91517941ccfbf4fb61b"}
Requesting team from public API...
Critical Error: ****: API Error: Response was empty
Critical Error: ****: API didn't find any team - Skipping Account
```
This results in my team surrendering. Do you know if this is a bug, or if it's maybe just the result of too much load on your server?
